api/notd/manager.py

```diff
@@ -145,6 +145,12 @@ async def update_token_metadata_deferred(self, registryAddress: str, tokenId: st
     async def update_token_metadata(self, registryAddress: str, tokenId: str, shouldForce: bool = False) -> None:
         return await self.tokenManager.update_token_metadata(registryAddress=registryAddress, tokenId=tokenId, shouldForce=shouldForce)
 
+    async def update_token_ownership(self, registryAddress: str, tokenId: str, shouldForce: bool = False) -> None:
```

Comment:
what calls this?

Reply:
the message processor

Reply:
well okay what calls the deferred function then?